### DISCUSSION TITLE
install python 3.12 instead of 3.13

### DISCUSSION THREAD

**Discussion by joskezelensky** [skycocker/chromebrew#10666] [Q&A]
how to install python 3.12 instead of 3.13?

**Comment by joskezelensky** [comment]
the reason i want to is because i need to use audioop (discord pip package), which is removed in 3.13

**Comment by uberhacker** [comment]
Did you know there is a lts version at https://pypi.org/project/audioop-lts/?

**Comment by joskezelensky** [comment]
no, thanks!
